### ISSUE TITLE
V5 Grid System docs issues

### ISSUE THREAD

**Issue by KohheePeace** [twbs/bootstrap#33670]
I've found docs issue at
https://getbootstrap.com/docs/5.0/layout/grid/#example

## 📌 Issue 
![2021-04-18_19h41_22](https://user-images.githubusercontent.com/29557494/115142763-7be71f80-a07e-11eb-8fa5-dceaf100f8c8.png)

Does `col-sm` creates three equal-width columns **across all devices and viewports**?

## 📌 Issue example
The below image is layout of iPhone 6 plus. **It doesn't create 3 columns.**
![2021-04-18_19h49_17](https://user-images.githubusercontent.com/29557494/115142927-3aa33f80-a07f-11eb-8991-a9990f563b1e.png)

## 📌 Docs at v4.6
I think docs at v4.6 is correct.
https://getbootstrap.com/docs/4.6/layout/grid/#how-it-works

> The above example creates three equal-width columns **on small, medium, large, and extra large devices** using our predefined grid classes. Those columns are centered in the page with the parent .container.

## PR
If this is an issue that needs to be fixed, I'd like to send a PR.

Thanks !

## Environment
Operating system and version: Windows
Browser and version: Chrome 89.0.4389.114

**Comment by ffoodd** [comment]
I agree, this is a wrong assertion: either get the 4.6 docs sentence back or get rid off `-sm` suffix to make the sentence true. The latter would require to reword columns content (maybe something like "⅓ column").

**Comment by KohheePeace** [comment]
I think it would be better to remove the `sm` prefix.

Because the following code is the simplest grid system example.
```html
<div class="container">
  <div class="row">
    <div class="col">
    </div>
    ...
  </div>
</div>
```

It also makes sense to separate this "example" section from the "How it works" section in v5.0

**Comment by mdo** [comment]
Closed by #33676
